packages/event-consumer/src/utilities/config.ts

```diff
@@ -22,15 +22,31 @@ const KafkaConfig = z
     KAFKA_BROKERS: z.string(),
     KAFKA_CLIENT_ID: z.string(),
     KAFKA_GROUP_ID: z.string(),
+    KAFKA_DISABLE_AWS_IAM_AUTH: z
+      .enum(["true", "false", "TRUE", "FALSE"])
+      .optional(),
   })
   .transform((c) => ({
     kafkaBrokers: c.KAFKA_BROKERS,
     kafkaClientId: c.KAFKA_CLIENT_ID,
     kafkaGroupId: c.KAFKA_GROUP_ID,
+    kafkaDisableAwsIamAuth:
+      c.KAFKA_DISABLE_AWS_IAM_AUTH === "TRUE" ||
+      c.KAFKA_DISABLE_AWS_IAM_AUTH === "true",
```

Comment:
may this work to be more generic?

```suggestion
    KAFKA_DISABLE_AWS_IAM_AUTH: z
      .string()
      .transform((value) => value.toUpperCase() === 'TRUE')
      .optional(),
  })
  .transform((c) => ({
    kafkaBrokers: c.KAFKA_BROKERS,
    kafkaClientId: c.KAFKA_CLIENT_ID,
    kafkaGroupId: c.KAFKA_GROUP_ID,
    kafkaDisableAwsIamAuth: c.KAFKA_DISABLE_AWS_IAM_AUTH,
```

Reply:
IMOH: 
this choice is related to how rigorous we want to be on Boolean values used in a environment variable, using enum make us more strict on values allowed otherwise more less.
If it's ok for you I suggest to be more strict with boolean values checking, that choice avoid any ambiguities about usages not other values allowed for a flag, and also, if values comes from some script or IAC is more desirable map value only with boolaen type values.

Reply:
Actually, for other config values, aren't we supporting lowercase only? Check `MOCK_FILE_MANAGER`. Why support uppercase here?

Reply:
@tpetrucciani I don't  why allowing only lowercase for `MOCK_FILE_MANAGER`, from my perspective it make this values more flexible so it's more time safety to avoid detecting and correct parsing error with env configurations. 
Anyway I don't know if was discussed a shared policy for env variables case sensitivity.
You suggest to adopt a strict UPPERCASE rule for evn variables?

Reply:
I don't think we discussed a policy explicitly, but I'd suggest keeping the same approach for all config variable values. So either always accept only lowercase for boolean values (as done for `MOCK_FILE_MANAGER` or in the other open PR for `SKIP_JWT_VERIFICATION`) or always accept both lowercase and uppercase as done here. I think it will be confusing if some variables can be set to either `true` or `TRUE` but others only work when set to `true`.

Reply:
ok, I'll follow current approach: only lowercase.